formatTest/typeCheckedTests/expected_output/attributes.re

```diff
@@ -267,20 +266,17 @@ class boxA ('a) (init: 'a) =
  * In non-curried sugar, the class_expr still sticks to "the simple thing".
  */
 class boxB ('a) (init: 'a) =
-  [@stillOnTheReturnBecauseItsSimple]
```

Comment:
I found the formatting change in this diff to be a slight improvement, especially if we were to allow omitting the brackets:

```
@attr {
  bracedExpression
}
```